appcheck/firebase-appcheck/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Unreleased
-
+* [unchanged] Update to internal interop API
```

Comment:
Maybe `[changed] Internal updates to allow Firebase SDKs to obtain limited-use tokens.` or something liked that?